pkg/graveler/graveler.go

```diff
@@ -191,6 +192,18 @@ type Repository struct {
 	StorageNamespace StorageNamespace `db:"storage_namespace"`
 	CreationDate     time.Time        `db:"creation_date"`
 	DefaultBranchID  BranchID         `db:"default_branch"`
+	State            RepositoryState
```

Comment:
I think this is the first occurrence that we use the protobuf type in the golang struct. Why did you remove the golang type?

Reply:
No need for redundant declaration of of enums - it will require unnecessary maintenance when adding / removing states. We only duplicate proto structs to keep backwards compatibility with DB. Optimally we would never have needed to create golang structs and perform the conversions all the time